### DISCUSSION TITLE
Bug report，how can i disable the detect in flagging-modules-as-esm

### DISCUSSION THREAD

**Discussion by Linyan-ma** [webpack/webpack#14102] [Q&A]
# Bug report

**What is the current behavior?**
I want to add third-party dependencies to babel processing。When referencing a third-party dependency, its internal package.json's type is module, according to https://webpack.js.org/guides/ecma-script-modules/#flagging-modules-as-esm, webpack transfer it as esmodules ,In the browser, Report an error: exports is not defined.

**If the current behavior is a bug, please provide the steps to reproduce.**

**What is the expected behavior?**
It displays normally in the browser

**Other relevant information:**
webpack version:  5.51.1
Node.js version: 16.4.2
Operating System: mac os
Additional tools:

**Comment by alexander-akait** [comment]
> If the current behavior is a bug, please provide the steps to reproduce.

Please fill out
